src/plugins/lua/antivirus.lua

```diff
@@ -23,46 +23,6 @@ local redis_params
 
 local N = "antivirus"
 
-if confighelp then
```

Comment:
Commits applied to the master branch likely pick apply cleanly to rspamd-1.5. Applying your changes to rspamd-1.5 and then reverse cherry-picking is less likely to work well (but could). Copying files wholesale between branches won't work.